runtime/tracing/tracing-core/common/src/aws/smithy/kotlin/runtime/tracing/DefaultTracer.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.smithy.kotlin.runtime.tracing
+
+import aws.smithy.kotlin.runtime.util.InternalApi
+
+@InternalApi
+public class DefaultTraceSpan(
```

Comment:
Does this need to be `public` or is it just an internal detail of `DefaultTracer`?